### PR TITLE
Update foo.php

### PR DESCRIPTION
**Pull request by muglug**

### CODE REVIEW

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm found errors in other files

```
src/foo.php:3:5 - Argument 1 of foo expects int, string(hello) provided
```

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm found errors in other files

```
src/foo.php:3:5 - Argument 1 of foo expects int, string(hello) provided
```

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm found errors in other files

```
src/foo.php:3:5 - Argument 1 of foo expects int, string(hello) provided
```

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm found errors in other files

```
src/foo.php:3:5 - Argument 1 of foo expects int, string(hello) provided
```

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm found errors in other files

```
src/foo.php:3:5 - Argument 1 of foo expects int, string(hello) provided
```

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm found errors in other files

```
src/foo.php:3:5 - Argument 1 of foo expects int, string(hello) provided
```

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm found errors in other files

```
src/foo.php:3:5 - Argument 1 of foo expects int, string(hello) provided
```

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
some test

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm found errors in other files

```
src/foo.php:3:5 - Argument 1 of foo expects int, string(hello) provided
```

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm found errors in other files

```
src/foo.php:3:5 - Argument 1 of foo expects int, string(hello) provided
```

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
dssadasf

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
(outdated review)

**Review by psalm-shepherd — CHANGES_REQUESTED** [review]
Psalm found errors in other files

```
src/foo.php:3:5 - Argument 1 of foo expects int, string(hello) provided
```

File: src/foo.php [hunk 1]
```diff
@@ -28,4 +28,4 @@
 
 
 
- function foo(string $s) : void {}
+ function foo(int $s) : void {}
```
Comment (psalm-spirit) [inline src/foo.php]: fdfsdfsd

Comment (psalm-shepherd) [inline src/foo.php]: fdsfss

**Review by psalm-shepherd — COMMENTED** [review]
gfaad

**Comment by psalm-shepherd** [comment]
is this a reviewfsa

**Comment by psalm-spirit** [comment]
Psalm found errors in other files

```
src/foo.php:3:5 - Argument 1 of foo expects int, string(hello) provided
```

**Comment by psalm-spirit** [comment]
Psalm found errors in other files

```
src/foo.php:3:5 - Argument 1 of foo expects int, string(hello) provided
```

File: src/foo.php [hunk 2]
```diff
@@ -1,31 +1,5 @@
...
-
-
-
-
-
-
-
-
- function foo(string $s) : void {}
+echo $b;
```
Comment (psalm-spirit) [inline src/foo.php]: Cannot find referenced variable $b in global scope

**Comment by psalm-spirit** [comment]
Psalm found some errors

**Comment by psalm-shepherd** [comment]
Psalm didn’t find any errors!

**Pull request closed without merging**